sympy/utilities/tests/test_autowrap.py

```diff
@@ -91,7 +91,7 @@ def test_cython_wrapper_inoutarg():
 def test_cython_wrapper_compile_flags():
     from sympy import Equality
     x, y, z = symbols('x,y,z')
-    routine = make_routine("test", Equality(z, x + y))
+    routine = [make_routine("test", Equality(z, x + y))]
```

Comment:
my earlier comment on not changing any tests applies here as well.

Reply:
The problem is that `_prepare_files` takes now a list of routines so I can move the construction of this list when `_prepare_files` is called.

Reply:
Does this break public API?

Reply:
Oh, my mistake, it was only used in `_prepare_files` - so I guess no.